include/seqan3/std/concept/callable.hpp

```diff
@@ -42,38 +42,74 @@
 #include <range/v3/range_concepts.hpp>
 #include <range/v3/utility/functional.hpp>
 
+#include <seqan3/std/concept/comparison.hpp>
+
 namespace seqan3
 {
 
 /*!\addtogroup concept
  * \{
  */
 
-/*!\brief Resolves to `ranges::Invocable<func, ...args>()`
- * \sa http://en.cppreference.com/w/cpp/experimental/ranges/concepts/Invocable
+/*!\interface   seqan3::invocable_concept <>
+ * \brief       Specifies whether the given callable is invocable with the given arguments.
+ * \sa          http://en.cppreference.com/w/cpp/experimental/ranges/concepts/Invocable
  */
-template <typename f, typename ...args>
-concept bool invocable_concept =                    static_cast<bool>(ranges::Invocable<f, args...>());
+//!\cond
+template <typename fun_t, typename ...arg_types>
+concept bool invocable_concept = requires(fun_t && f, arg_types &&... args)
+{
+    ranges::invoke(std::forward<fun_t>(f), std::forward<arg_types>(args)...);
```

Comment:
why do you need to use ranges::invoke?

Reply:
There seem to be minor differences and I wanted to make sure that the concept behaves the same.